charts/temporal-operator/templates/serving-cert.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: cert-manager.io/v1
```

Comment:
Same comment as for issuer

Reply:
`certissue: true` also makes it optional.

Reply:
What do you think about `.Values.webhook.certManager.certificate.enabled` ? with a key to specify which kind of issuer we want to use ?